infra/terraform/terraform.go

```diff
@@ -199,12 +199,16 @@ func (r *terraform) Destroy(ctx context.Context) error {
 	}
 
 	varsPath := filepath.Join(r.stateDir, tfVarsFile)
-	_, err := r.command(ctx, []string{
+	destroyCommand := []string{
 		"destroy", "-force",
 		"-var", fmt.Sprintf("nodes=%d", r.NumNodes),
 		"-var", fmt.Sprintf("os=%s", r.OS),
 		fmt.Sprintf("-var-file=%s", varsPath),
-	})
+	}
+	if r.CustomVarsFile != "" {
+		destroyCommand = append(destroyCommand, fmt.Sprintf("-var-file=%s", r.CustomVarsFile))
```

Comment:
Is it intentional that -var-file get's passed twice? Once with varsPath and once with r.CustomVarsFile?

Reply:
Yes. `varsPath` contains variables which we get from robotest config, like aws user/pass and etc. `r.CustomVarsFile` provides a way to add custom variables to terraform command, without adding all that variables into robotest config.